woocommerce/woocommerce-android
# Fix coupon button visibility in order edition flow

### Description

This PR fixes the initial state of the "coupon button" in the Order **edition** flow. The button is disabled by default unless we add the product. However, in order edition flow it should be enabled if the order already contains at least one product.

### Testing instructions

1. Create an order with a product.
2. Go back to the order and click "Edit"
3. Verify it's possible to add a coupon to the order.

### Images/gif

| Before  | After |
| ------------- | ------------- |
| ![before](https://github.com/woocommerce/woocommerce-android/assets/4527432/7292f3ce-6cc9-4ea4-b56a-c8d39345fa50) | ![after](https://github.com/woocommerce/woocommerce-android/assets/4527432/3c185308-f45b-4e65-b79a-c587a1b379b3)  |
